### DISCUSSION TITLE
Is bevel support deploy fabric with prometheus on k8s?

### DISCUSSION THREAD

**Discussion by SamYuan1990** [hyperledger-labs/blockchain-automation-framework#1891] [Q&A] [answered]
Hi Bevel,

ref to https://github.com/hyperledger-labs/PerformanceSandBox/blob/main/doc/SUT.md#deployment-intergration
I wanna to know if bevel is able to support deploy fabric with prometheus on k8s?

if yes, please let me know which doc/code can I reference with?

thanks and regards
Sam

**Comment by sownak** [comment, marked as answer]
There is an open issue #1416 for Promethues set up using Bevel, but it is not a priority as Prometheus can be deployed using multiple other ways. If you need the ports to be enabled on Fabric nodes, please create a feature request for that and you can work on that as well.

**Reply by SamYuan1990** [reply]
Bevel as far as I know is a blockchain automation/deployment tool right?
so I am agree that Prometheus is able to be deployed in anyway.

I will create an issue as feature request for fabric nodes.
but I am not sure if I have bandwidth work on it or not.

May I know if Bevel had discussed enable Fabric nodes with Prometheus or not?
Any plan for integration Fabric with Prometheus in Bevel?

**Reply by sownak** [reply]
@SamYuan1990 No plans from the core team as such because this is the only request we have got. If you can create an Issue and that gets a heads-up or if anyone from community picks that up, we will consider it in the next sprint.
